### PR TITLE
Investigate and fix CI/CD Pipeline

### PR DESCRIPTION
**Pull request by sahilsaini432**
**Changes Made**

- Updated verifyImage endpoint to not save images anymore.
- Updated service docker image to be `buster-slim` instead of `alpine`
- Updated service docker file to use `ecosystem.config.js` file to start the service
- Updated server to use 2600MB memory
- Updated postimage endpoint to compress images before uploading to the S3 bucket. (now every image is saved as a jpeg).
- Removed https://github.com/Dubble-bubble-original/engg4000/blob/dev/Automation/src/test/java/api/image.feature#L71-L76 Automated API test, this won't pass anymore because the images are compressed.

### CODE REVIEW

File: nginx/configs/default.conf [hunk 1]
```diff
@@ -16,12 +16,12 @@ server {
 
     location / {
         proxy_pass http://client/;
-        client_max_body_size 2M;
+        client_max_body_size 1G;
```
Comment (nathanielcaron) [inline nginx/configs/default.conf:19]: This setting is for the request body size:
http://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size

1G seems outrageously high 😄

Comment (sahilsaini432) [reply nginx/configs/default.conf:19]: Ya, I was just trying something out 😄

**Review by nathanielcaron — APPROVED** [review]

**Comment by nathanielcaron** [comment]
We did it! 🥳 
Great work @sahilsaini432

**Review by fredy20501 — APPROVED** [review]
Woo! 🥳 🔥 🎆

**Review by JoaoPontes98 — APPROVED** [review]
You the man!

**Review by CarterMacLennan — APPROVED** [review]

**Pull request merged**